cpu/kinetis/dist/check-fcfield.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh
+#
+# Flash configuration field check script for Freescale Kinetis MCUs.
+#
+# This script is supposed to be called from RIOTs
+# unified OpenOCD script (dist/tools/openocd/openocd.sh).
+#
+# Syntax: check-fcfield.sh $FLASHFILE
+#
+# @author       Jonas Remmert <j.remmert@phytec.de>
+# @author       Johann Fischer <j.fischer@phytec.de>
+# @author       Joakim Nohlgård <joakim.nohlgard@eistec.se>
+# @author       Francisco Molina <francisco.molina@inria.fr>
+
+if [ $(printf '%d' "${IMAGE_OFFSET}") -gt 1040 ] ; then
+    echo "WARN: we don't check the fcfield value if flashing at \$IMAGE_OFFSET > 0x410"
+    exit 0
+fi
+
+if [ $# -ne 1 ]; then
+    echo "Usage: $0 FLASHFILE"
+    echo "Checks the flash configuration field protection bits to avoid flashing a locked image to a Kinetis MCU (protection against accidental bricking)."
+    exit 2
+fi
+
+FLASHFILE="$1"
+
+FCFIELD_START='0x400'
+FCFIELD_END='0x410'
+FCFIELD_AWK_REGEX='^ 0400 '
+
+if [ ${FLASHFILE##*.} = bin ]; then
```

Comment:
I would prefer adding the `.bin` and `IMAGE_OFFSET` in a second commit, to show "this was the old behavior moved in the same file`, `this is adding `.bin` support.